packages/app/src/app/overmind/actions.ts

```diff
@@ -354,10 +359,22 @@ export const signInVercelClicked = async ({
   if (data && data.code) {
     try {
       state.user = await api.createVercelIntegration(data.code);
+    } catch (error) {
+      actions.internal.handleError({
+        message: 'Not able to add a Vercel integration. Please try again.',
+        error,
+      });
+    }
+
+    try {
       await actions.deployment.internal.getVercelUserDetails();
+
+      // Not sure if we ever reach the catch clause below because the error has already
+      // been caught in getVercelUserDetails.
     } catch (error) {
       actions.internal.handleError({
-        message: 'Could not authorize with Vercel',
+        message:
+          'We were not able to fetch your Vercel user details. You should still be able to deploy to Vercel, please try again if needed.',
```

Comment:
Could we add this message to a constant?

Reply:
Yeah that's not a bad idea. I spent some time looking but I can't really find a good file to add it to right now, and I don't want to create a new file for just this string. Do we want a task to extract more consts like these?